### PR TITLE
Add support for performance evaluation via a Wrapper

### PR DESCRIPTION
**Pull request by harperj**
This change adds a gym wrapper for collecting max rewards and max floors
across a set of seeds provided by the user.  Includes an update to the
README and an example of usage which mimics OTC evaluation.

### CODE REVIEW

File: examples/evaluation.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import multiprocessing
+
+def run_episode(env):
+    done = False
+    reward = 0.0
+    
+    while not done:
+        action = env.action_space.sample()
+        obs, reward, done, info = env.step(action)
+    return reward
```
Comment (awjuliani) [inline examples/evaluation.py]: This reward is just the last one received, not the return for the episode.

File: examples/evaluation.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    # the "evaluation_complete" flag is True.  Attempting to step or reset after
+    # all of the evaluation seeds have completed will result in an exception.
+    while not env.evaluation_complete:
+        episode_rew = run_episode(env)
+
+    # Finally the evaluation results can be fetched as a dictionary from the 
+    # environment wrapper.
+    print(env.results)
+
+    env.close()
```
Comment (awjuliani) [inline examples/evaluation.py]: Add newline

File: obstacle_tower_env.py [hunk 3]
```diff
@@ -407,3 +390,98 @@ def lookup_action(self, action):
...
+    """
+    Environment wrapper for performing evaluation. Accepts an ObstacleTowerEnv and a list 
+    of seeds and will collect resulting rewards and floors reached for each seed.  This wrapper 
+    automatically resets the environment, so an external environment reset is not necessary.
+    """
+
+    def __init__(self, env, seeds):
+        """
+        Arguments:
+        env: ObstacleTowerEnv for
```
Comment (awjuliani) [inline obstacle_tower_env.py]: This docstring seems incomplete.

File: examples/evaluation.py (hunk 1 above)
Comment (awjuliani) [reply examples/evaluation.py]: Addressed it in a commit.

File: examples/evaluation.py (hunk 2 above)
Comment (awjuliani) [reply examples/evaluation.py]: Addressed it in a commit.

File: obstacle_tower_env.py (hunk 3 above)
Comment (awjuliani) [reply obstacle_tower_env.py]: Fixed.

File: examples/evaluation.py (hunk 1 above)
Comment (harperj) [reply examples/evaluation.py]: 🤦‍♂

**Comment by harperj** [comment]
Oh, whoops -- I had already fixed a couple of these things and not pushed the changes.  Thanks for making the fixes.  I will make the reset parameters change and review.

**Comment by harperj** [comment]
Rebased and added reset parameters.  PTAL.

**Pull request merged**